test/hotspot/jtreg/runtime/interpreter/LastJsrTest.java

```diff
@@ -24,17 +24,22 @@
 /*
  * @test
  * @bug 8335664
+ * @bug 8338924
  * @summary Ensure a program that ends with a JSR does not crash
  * @library /test/lib
  * @compile LastJsr.jasm
  * @compile LastJsrReachable.jasm
- * @run main/othervm LastJsrTest
+ * @run main/othervm
+ *      -Xbatch
+ *      LastJsrTest
  */
 
 public class LastJsrTest {
     public static void main(String[] args) {
-        LastJsr.test();
-        LastJsrReachable.test();
+        for (int i = 0; i < 1000; ++i) {
```

Comment:
Don't you need 10,000 in your loop to trigger compilation?

Reply:
Yes for C2, but this is enough for C1, the only compiler that needs this fix.  I wanted to make sure C1 compilation was triggered by default without -Xcomp.  Testing tiers that use -Xcomp will make sure it passes with C2.

Reply:
Ok.